resources/qml/PrintSetupSelector/Recommended/RecommendedSettingItem.qml

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2022 UltiMaker
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 2.10
+
+import UM 1.5 as UM
+import Cura 1.7 as Cura
+
+
+Item
+{
+    id: settingItem
+    width: parent.width
+    Layout.minimumHeight: UM.Theme.getSize("section_header").height
+    Layout.fillWidth: true
+
+    property alias settingControl: settingContainer.children
+    property alias settingName: settingLabel.text
+    property string tooltipText: ""
+    property bool isCompressed: false
+
+    UM.Label
+    {
+        id: settingLabel
+        width: leftColumnWidth
+        anchors.left: parent.left
+        anchors.verticalCenter: parent.verticalCenter
+        // These numbers come from the IconWithText in RecommendedSettingSection
+        anchors.leftMargin: UM.Theme.getSize("medium_button_icon").width + UM.Theme.getSize("default_margin").width
+    }
+
+    MouseArea
+    {
+        id: tooltipArea
+        anchors.fill: settingLabel
+        propagateComposedEvents: true
+        hoverEnabled: true
+        onEntered: base.showTooltip(parent, Qt.point(-UM.Theme.getSize("thick_margin").width, 0), tooltipText)
+        onExited: base.hideTooltip()
+    }
+
+
+    Loader
```

Comment:
Should this be a `Loader` element? You are aliassing the children using
```
property alias settingControl: settingContainer.children
```
You're not using the `Loader`s source property, I think this can be substitutted with an `Item` component.